app/presenters/protected_area_presenter.rb

```diff
@@ -161,7 +161,7 @@ def num_fields_with_data
 
   def parse_management_plan management_plan
     if (management_plan.is_a? String) && (management_plan.starts_with?("http"))
-      link_to("View Management Plan", management_plan)
+      ActionController::Base.helpers.link_to(I18n.t('stats.management-plan.button'), management_plan, title: I18n.t('stats.management-plan.button-title', site: @protected_area.name))
```

Comment:
should it be target blank? If you're editing this, can it be split onto multiple lines to make more legible?